tools/etcd-top/etcd-top.go

```diff
@@ -0,0 +1,213 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"flag"
+	"fmt"
+	"math"
+	"net/http"
+	"os"
+	"runtime"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/akrennmair/gopcap"
```

Comment:
we rewrite the import path for other component inside etcd repo. So I think we need to do this for etcd-top too. So go build will just work.